lib/getstats.js

```diff
@@ -70,14 +70,18 @@ function _getStats(peerConnection, options) {
 function getActiveIceCandidatePairStats(peerConnection, options) {
   options = options || {};
 
-  if (typeof options.testForChrome !== 'undefined'
-    || typeof webkitRTCPeerConnection !== 'undefined') {
+  var browser = guessBrowser();
+
+  if (browser === 'chrome' || typeof options.testForChrome !== 'undefined') {
```

Comment:
Where are these `testForChrome`, etc., flags defined and used? If they are opt-in, you don't need to check `typeof`. You can instead just write, e.g.

```js
if (browser === 'chrome' || options.testForChrome) {
  // ...
}
```

Reply:
Those are the flags which we have been using for unit testing. Yes they are opt-in. I have removed typeof check